src/types.ts

```diff
@@ -255,7 +258,9 @@ export type ContractEventArgs_3_0_0 =
   | BlacklistTransferManagerEventArgs_3_0_0
   | VestingEscrowWalletEventArgs_3_0_0;
 
-export type ContractEventArgs = ContractEventArgs_3_0_0;
+export type ContractEventArgs_3_1_0 = RestrictedPartialSaleTMEventArgs_3_1_0;
```

Comment:
I'm guessing you intended to add `RestrictedPartialSaleTMEventArgs_3_1_0` to the union type but instead you replaced this

Reply:
This is the code I am using now:

```
export type ContractEventArgs_3_1_0 = RestrictedPartialSaleTMEventArgs_3_1_0;
export type ContractEventArgs = ContractEventArgs_3_0_0 | ContractEventArgs_3_1_0;
```
Let me know if thats incorrect

Reply:
Kinda, but it's kinda my fault as well, I forgot to declare `ContractEventArgs_3_1_0` before

`ContractEventArgs_3_1_0` should be a union type of all the event args for the contracts that are represented in `ContractEvents_3_1_0`, not just the RPSTM

But yeah, you're on the right path

Reply:
At the time there was no other 3_1 contract event args

Reply:
Voting will be apart of this as well